README.md

```diff
@@ -1,12 +1,79 @@
-# template-project
+Thoth's TensorFlow stack guidance example
+-----------------------------------------
 
-This is a Template for any Python based project, it contains what Project Thoth and the AI CoE need:
+**See different branches for different examples**
 
-1. GitHub defaults and Templates for issues
-2. configuration for Coala and Black (code formating)
-3. basic configuration for Zuul
-4. configuration for Thoth (stage environment, Red Hat VPN only)
-5. if you are writing a Python module, Kebechet could manage releases of your packages for you
-6. if credentials are provided, Python module releases could be published to PyPI
+This is an example of an application which uses Thoth's recommendations to
+recommend a TensorFlow stack for a specific hardware. The application is
+showing one of the `Integration of Thoth
+<https://pypi.org/project/thamos>`_ using Thamos CLI.
 
-Dependencies should be managed using `pipenv` (`Pipfile`, and the `Pipfile.lock` could be created by `thamos advise`), `pip3` and a `requirements.txt` files could be used.
+Thamos CLI
+==========
+
+One of the integration for Thoth is `Thamos
+<https://pypi.org/project/thamos>`_. You can use Thoth's recommendation engine
+directly from within your terminal. First, you need to clone this repo and
+install Thamos CLI:
+
+.. code-block:: console
+
+  git clone https://github.com/thoth-station/cli-example.git && cd cli-example
+  pip3 install thamos
+  thamos --help
+
+The pre-configured template for Thamos CLI is available in the
+``.thoth.yaml`` file:
+
+.. code-block:: console
+
+  cat .thoth.yaml
+
+Alternatively, to generate Thoth's configuration file out of the template run the
+following command:
+
+.. code-block:: console
+
+  thamos config --no-interactive --template thoth_conf_template.yaml
```

Comment:
The config file expansion based on template is more suitable for OpenShift's s2i. `thamos config` should work just fine for setup based on the local environment.

Reply:
Hm, but without the template we don't have any control on what environment the user runs... so let's keep it...